beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -179,14 +178,16 @@ func (vs *Server) GetBeaconBlock(ctx context.Context, req *ethpb.BlockRequest) (
 	}
 	sBlk.SetStateRoot(sr)
 
-	fullBlobs, err := blobsBundleToSidecars(blobBundle, sBlk.Block())
+	fullBlobs, err := blobsBundleToSidecars(fullBlobsBundle, sBlk.Block())
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "Could not convert blobs bundle to sidecar: %v", err)
 	}
-	blindBlobs, err := blindBlobsBundleToSidecars(blindBlobBundle, sBlk.Block())
+	fullBlobsBundle = nil // Reset full blobs bundle.
```

Comment:
should we also set it to nil if it errors using it above?

Reply:
yep 26a93c52c